packages/dashboard/src/components/DashboardGridLayout/examples/DGLWithAddCardPanel.Example.tsx

```diff
@@ -0,0 +1,285 @@
+import * as React from 'react';
+import {
+  IDGLCard,
+  ICardContentDetails,
+  Priority,
+  CardContentType,
+  CardSize,
+  DashboardGridLayoutWithAddCardPanel,
+  DashboardGridBreakpointLayouts,
+  IThumbnailItemProps
+} from '../../../index';
+
+// Cards information that go into the layout and add card panel
+const cardFrameContent = {
+  cardTitle: 'Example Card',
+  cardDropDownOptions: [
+    {
+      key: 'Remove',
+      name: 'Remove',
+      icon: 'PageRemove',
+      ariaLabel: 'Remove card',
+      title: 'Remove card',
+      onClick: () => {
+        alert('Remove clicked');
+      }
+    }
+  ]
+};
+
+const thumbnailItems: IThumbnailItemProps[] = [
+  {
+    imageSource: './src/images/download.jpg',
+    subheaderText: 'First item',
+    description: 'This is the first thumbnail item',
+    handleThumbnailItemClick: () => {
+      alert('First Item clicked');
+    }
+  },
+  {
+    imageSource: './src/images/download.jpg',
+    subheaderText: 'Second item',
+    description: 'Lorem ipsum dolor sit amet, ',
+    handleThumbnailItemClick: () => {
+      alert('Second Item clicked');
+    }
+  }
+];
+
+const getHeader = (name: string) => {
+  return {
+    headerText: 'Card ' + name,
+    annotationText: 'Annotation Text '
+  };
+};
+
+const contentAreaList: ICardContentDetails[] = [
+  {
+    priority: Priority.Priority1,
+    cardContentType: CardContentType.BodyText,
+    content: {
+      subHeaderText: 'Subheader Text',
+      bodyText: 'Body Text'
+    }
+  },
+  {
+    priority: Priority.Priority2,
+    cardContentType: CardContentType.ThumbnailList,
+    content: {
+      thumbnailItems: thumbnailItems
+    }
+  }
+];
+const cardsVisibleInLayout: IDGLCard[] = [
+  {
+    id: '0',
+    cardFrameContent: cardFrameContent,
+    header: getHeader('0'),
+    cardContentList: contentAreaList,
+    cardSize: CardSize.small,
+    x: 0,
+    y: 0,
+    addCardInfo: {
+      addCardPanelBodyText:
+        'Add and manage users in your organization, help reset their passwords, manage their licenses,' +
+        ' and see their latest Azure AD Connect sync.',
+      addCardPanelHeader: 'User Management',
```

Comment:
>User Management [](start = 27, length = 15)

use Lorem

Reply:
Changed every string on example page to Lorem Ipsum.